website/docs/rbi.md

```diff
@@ -181,6 +181,13 @@ the source folders for every gem in a project. If any gems have a top-level
 `rbi/` or `rbis/` folder, `srb` will collect the paths to any such folders into
 a file (`sorbet/rbi_list`) and mention this file in the `sorbet/config` file.
 
+If you add a gem that have a `rbi/` or `rbis/` folder, rerunning `sorbet-typed` 
+command will update the `sorbet/rbi_list`.
+
+```bash
+srb rbi sorbet-typed
+```
+
```

Comment:
Is this correct for the most recent version of `srb init`? I think we removed `sorbet/rbi_list` in favor of computing + caching effectively the same list before `srb tc`.

@manhhung741 Can you check that you're on the latest version of Sorbet?

Reply:
These changes were first introduced in `0.4.4137` but there were a number of bug fixes that didn't land until `0.4.4255`.

Reply:
Oh? I didn't know that you remove `sorbet/rbi_list`? It'd be great if you inform us these kinds of changes.

Reply:
Anyway, here is the context for this PR. Maybe you have a different answer now.

https://stackoverflow.com/questions/56741667/how-to-use-specific-rbi-file-with-tc-command/56741987#56741987

Reply:
@jez I just tested on `sorbet (0.4.4239)` and it's not working. Removing the `rbi_config` file and sorbet stops including `rbis/` from my gem